### PR TITLE
Adds paragraph describing a product I would like to build in the next 5 years.

### PR DESCRIPTION
**Pull request by b-carter-allen**
This PR adds a paragraph element filled with words that mean things.

### CODE REVIEW

**Comment by b-carter-allen** [comment]
Rainbow effects were added.
image

File: src/carter-mc-barter-narter-is-dope.css [hunk 1]
```diff
@@ -1,7 +1,44 @@
...
+.main-page-title {
+  text-align: center;
+  min-height: 5vh;
+  font: roboto;
+  font-size: 24px;
+}
+
+.main-page-title:hover {
+  font-size: 24px;
+  text-align: center;
```
Comment (cchanningallen) [inline src/carter-mc-barter-narter-is-dope.css]: this shouldn't be necessary

File: src/carter-mc-barter-narter-is-dope.css [hunk 2]
```diff
@@ -1,7 +1,44 @@
...
+  background-image: linear-gradient(to left, violet, indigo, blue, green, yellow, orange, red);
```
Comment (cchanningallen) [inline src/carter-mc-barter-narter-is-dope.css]: Still want this, now that you have `.rainbow-text`?

File: src/carter-mc-barter-narter-is-dope.css [hunk 3]
```diff
@@ -1,7 +1,44 @@
...
+  -webkit-background-clip: text;
+  color: transparent;
+  position: relative;
```
Comment (cchanningallen) [inline src/carter-mc-barter-narter-is-dope.css]: Why is this necessary?

**Review by cchanningallen — APPROVED** [review]
DOPESAUCE
